.github/workflows/pr.yml

```diff
@@ -25,4 +20,6 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-gradle-
     - name: Build
-      run: ./gradlew -PenableCrossCompilerPlugin=true build
+      run: ./gradlew build
+    - name: Integration Tests
```

Comment:
Were lines 24 & 25 not meant to be part of this commit?

Reply:
Oops, there was a merge conflict and I got confused and resolved it backwards! Thanks for catching that!